src/TMedia.h

```diff
@@ -92,7 +92,7 @@ class TMediaData
 class TMediaPlayer
 {
 public:
-    TMediaPlayer() {}
+    TMediaPlayer() : mpHost(nullptr), mMediaPlayer(nullptr), initialized(false) {}
     ~TMediaPlayer() {}
 
     TMediaPlayer(Host* pHost, TMediaData& mediaData)
```

Comment:
*This* constructor would be better written, I believe, as:
```cpp
TMediaPlayer(Host* pHost, TMediaData& mediaData)
: mpHost(phost)
, mMediaData(mediaData)
, mMediaPlayer(new QMediaPayer(pHost))
, initilalized(true)
{}
```

Also, once this is done, `initialized` is properly initialized in BOTH constructors so it is pointless to use one of the later C++ enhancements that assigns an initialization (the ` = false` where it is declared under the `private:` heading) to it!

Reply:
I'm not sure you realised that I had attached this comment to the **second** constructor and not the first (the default one - with no arguments)...! 😜

Reply:
Seems like. Oh well.